charts/controlplane/templates/auth-z/route.yaml

```diff
@@ -0,0 +1,30 @@
+{{- $gatewayName := include "controlplane.name" . -}}
+{{- $name := include "controlplane.authZName" . -}}
+{{- $svcPort := .Values.authZ.service.httpPort -}}
+---
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: {{ $gatewayName }}-auth-z
```

Comment:
Warum ist authz hier auf einmal auseinander?

Reply:
Hab es nachgemacht, wie es bei den anderen services auch ist.

Reply:
Renamed this and a few other places to authz